src/transformers/modeling_tf_utils.py

```diff
@@ -882,7 +882,15 @@ def compute_loss(self, *args, **kwargs):
 
     def train_step(self, data):
         """
-        A modification of Keras's default train_step that cleans up the printed metrics when we use a dummy loss.
+        A modification of Keras's default `train_step` that cleans up the printed metrics when we use a dummy loss. If a
```

Comment:
Docs upgrade should ideally go in a separate PR, it would be best to keep this one focused on its purpose :-)

Reply:
Moved to a separate PR 👉  https://github.com/huggingface/transformers/pull/15755